docs/docs-beta/docs/guides/asset-checks.md

```diff
@@ -1,64 +1,74 @@
 ---
-title: "Test assets with Asset Checks"
+title: "Testing assets with Asset Checks"
 sidebar_position: 10
+sidebar_label: "Asset Checks"
 ---
 
-Asset checks in Dagster provide a way to define and execute different types of data quality checks on your data assets directly in Dagster.
+Asset checks in Dagster provide a way to define and execute different types of data quality checks on your data assets.
 
 This guide covers the most common use cases for testing assets with asset checks and taking action based on the result.
 
 <details>
 <summary>Prerequisites</summary>
+
+To follow this guide, you'll need:
+
 - Familiarity with [Assets](/concepts/assets)
 </details>
 
```

Comment:
I think asset checks is a deep enough topic that it warrants a short overview, maybe something like this?

```suggestion
## Getting Started with Asset Checks

To get started with asset checks, you can follow these general steps, which are detailed below:

1. **Define an asset check:** Asset checks are typically defined using the `@asset_check` or `@multi_asset_check` decorator and run either within an asset or separate from the asset.
2.** Pass the asset checks to the `Definitions` object:** Asset checks must be added to Definitions for Dagster to recognize them.
3. **Choose how you wish to execute asset checks**: by default, all jobs targeting an asset will also run associated checks, although you can run asset checks through the UI.
4. **View asset check results in the UI**: Asset check results will appear in the UI and can be customized through the use of metadata and severity levels
5. **Alert on failed asset check results**: If you are using Dagster+, you can choose to alert on asset checks.

```